be/src/vec/columns/column.h

```diff
@@ -246,6 +246,14 @@ class IColumn : public COW<IColumn> {
     /// Returns pointer to the position after the read data.
     virtual const char* deserialize_and_insert_from_arena(const char* pos) = 0;
 
+    virtual size_t get_max_row_byte_size() const { return 0; }
```

Comment:
Add some comments for new method. Then other people could read the code more clearly.